### PR TITLE
Clarify what autogenerate compares

### PR DESCRIPTION
**Pull request by peterjc**
### Description

I was struggling with empty upgrade/downgrade functions because both my database and my ORM were up to date. I wrongly assumed the comparison was against the previous database scheme as per the prior revision script(s).

### Checklist

This pull request is:

- [x] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

**Comment by peterjc** [comment]
Cross reference https://github.com/sqlalchemy/alembic/discussions/1086

File: docs/build/autogenerate.rst [hunk 1]
```diff
@@ -1,10 +1,12 @@
 Auto Generating Migrations
 ===========================
 
-Alembic can view the status of the database and compare against the table metadata
-in the application, generating the "obvious" migrations based on a comparison.  This
-is achieved using the ``--autogenerate`` option to the ``alembic revision`` command,
-which places so-called *candidate* migrations into our new migrations file.  We
+Alembic can view the status of the database (pointed to by ``sqlalchemy.url`` in
+your ``alemic.ini`` file using the *previous* schema) and compare against the
```
Comment (zzzeek) [inline docs/build/autogenerate.rst:5]: "current" schema

File: docs/build/autogenerate.rst [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+table metadata in the application (your ORM which defines the *latest* schema),
```
Comment (zzzeek) [inline docs/build/autogenerate.rst:6]: "proposed" schema

**Review by zzzeek — CHANGES_REQUESTED** [review]
I think the term "previous" and "latest" are confusing, since these terms are just being introduced here without much context

**Comment by peterjc** [comment]
Your suggested terminology is clear to me too; updated.

**Review by zzzeek — APPROVED** [review]

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **zzzeek** to try to get revision dec968b136a813d27e1d2bca04574a2399343a1a of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change dec968b136a813d27e1d2bca04574a2399343a1a: https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5588

**Comment by sqla-tester** [comment]
**Federico Caselli** (CaselIT) wrote:

not 100% correct, but I guess it's generally right

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5588

**Comment by sqla-tester** [comment]
**Michael Bayer** (zzzeek) wrote:

agree close enough thanks for the review

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5588

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5588 has been **merged**. Congratulations! :)

**Pull request closed without merging**